### DISCUSSION TITLE
How to avoid unsupported bumps php composer

### DISCUSSION THREAD

**Discussion by firecow** [renovatebot/renovate#19311] [Request Help] [answered]
Hello, I'm getting MR's to packages that doesn't work with my projects expected PHP version.

![image](https://user-images.githubusercontent.com/973602/206420308-e99fea9b-17e7-42c8-9b0b-22d8a2859b6d.png)

`composer.json`
```
{
  "require-dev": {
    "php": "~7.4",
    "orchestra/testbench": "6.25.1"
  }
}
```

https://packagist.org/packages/orchestra/testbench#v7.15.0 Does not support php `~7.4`

Is there any way to avoid getting updates for packages that does not support the projects minimum php version?

**Comment by rarkins** [comment, marked as answer]
Unfortunately not yet. Please follow and/or comment on #2355
